test/server/server_test.cc

```diff
@@ -679,6 +679,17 @@ TEST_P(ServerInstanceImplTest, ShutdownBeforeWorkersStarted) {
   server_thread->join();
 }
 
+TEST_P(ServerInstanceImplTest, V2ConfigOnly) {
+  options_.service_cluster_name_ = "some_cluster_name";
+  options_.service_node_name_ = "some_node_name";
+  try {
+    initialize("test/server/test_data/server/unparseable_bootstrap.yaml");
+    FAIL();
+  } catch (const EnvoyException& e) {
+    EXPECT_THAT(e.what(), HasSubstr("Unable to parse JSON as proto"));
+  }
+}
+
```

Comment:
is it possible some unrelated stuff has comingled with this PR?

Reply:
fixed.